src/components/Academics/AcdemicExcellence.jsx

```
@@ -4,73 +4,84 @@ const excellenceData = [
-          </h1>
-        </div>
+      <div className="flex flex-col mt-[60px] gap-[35px]">
+        <h1 className="text-[45px] tracking-wide-30 font-bold text-[#184C77] ">Academic Excellence</h1>
+        <p className="text-[19px] text-[#184C77] tracking-wide-20  ">Our students thrive in an environment that values<br/>
+           curiosity, encourages independent thinking , and <br/>
+           nurtures a love for learning.</p>
+           <div className="flex flex-col gap-[20px]">
+           {[0, 1].map((cardIdx) => {
+                       
+                       return (
+                         <AnimatePresence mode="wait" >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)